elastic/kibana
# Filter out error docs from standalone cluster lists

## Summary

Ensures error docs won't lead to erroneous "Standalone cluster" entries.

Fixes https://github.com/elastic/integrations/issues/4011

Requires ~https://github.com/elastic/kibana/pull/137904~ and ~https://github.com/elastic/integrations/pull/4147~ to test functionality.

~Keeping as draft until those are merged so we can test properly.~ All merged, just make sure you have main kibana & packages.

### Testing

(Details WIP)

- Set up elastic-package stack with a logstash profile
- Make sure that profile has a bad hostname (to make sure you get error docs)
- Ensure no "Standalone Cluster" is present in Stack Monitoring UI
